### PR TITLE
Added in ability to specify a content_width for ColumnViews

### PR DESCRIPTION
**Pull request by bgrayson-mips**

### CODE REVIEW

File: helios/pipeViewer/scripts/alf_gen/ALFLayout.py
Comment (klingaard) [inline helios/pipeViewer/scripts/alf_gen/ALFLayout.py:269]: To allow backward compatibility, can you swap the order of the new param with time_offset?

**Review by bdutro — APPROVED** [review]

**Review by klingaard — APPROVED** [review]

**Comment by bgrayson-mips** [comment]
I can't merge this, since I don't have write access. Could one of you two do it? Thx.

**Comment by bdutro** [comment]
Done.

**Pull request merged**
